app/models/ext_management_system.rb

```diff
@@ -302,6 +303,8 @@ def validate_zone_not_maintenance_when_ems_enabled?
   supports_attribute :feature => :cloud_subnet_create
   supports_attribute :feature => :cloud_volume
   supports_attribute :feature => :cloud_volume_create
+  supports_attribute :feature => :cloud_volume_snapshots
+  supports_attribute :feature => :cloud_volume_snapshots_create
```

Comment:
Do we need `cloud_volume_snapshots_create` on the EMS?  I'd think that we would check for an EMS that supports cloud_volume_snapshots, then go to the cloud_volume_snapshot class to check if it supports create

Reply:
Yeah, actually we don't need it since the creation method is implemented in CloudVolume class. removing...